doc/classes/Callable.xml

```diff
@@ -120,6 +139,15 @@
 				Calls the method represented by this [Callable]. Unlike [method call], this method expects all arguments to be contained inside the [param arguments] [Array].
 			</description>
 		</method>
+		<method name="create" qualifiers="static">
+			<return type="Callable" />
+			<param index="0" name="variant" type="Variant" />
+			<param index="1" name="method" type="StringName" />
+			<description>
+				Creates a new [Callable] for the method named [param method] in the specified [param variant]. To represent a method of a built-in [Variant] type, a custom callable is used (see [method is_custom]). If [param variant] is [Object], then a standard callable will be created instead.
+				[b]Note:[/b] In GDScript, you can use [code]instance.method[/code] syntax in most cases. The exception is [Dictionary], since dot syntax is used to access keys. Also, you can use this method when the instance type is not known in advance (for polymorphism).
```

Comment:
Perhaps? (If I understand the method right?)
```suggestion
				[b]Note:[/b] In GDScript, to get or create a [Callable] you can use the dot syntax ([code]instance.method[/code]). This method is always needed for the [Dictionary] type, as dot syntax is used to access its entries. You may also use this method when [param variant]'s type is not known in advance (for polymorphism).
```
or (but I'm not sure since it loses information).
```suggestion
				[b]Note:[/b] This method is always necessary for the [Dictionary] type, as dot syntax is used to access its entries. You may also use this method when [param variant]'s type is not known in advance (for polymorphism).
```